### ISSUE TITLE
Error handling needed when checking nested groups

### ISSUE THREAD

**Issue by MagerValp** [munkireport/security#1]
We have machines that were previously AD bound, and had an AD group nested into `com.apple.local.ard_admin`. When the client is unbound from AD and no group exists with the corresponding UUID, this code crashes as the `dscl` output is empty:

https://github.com/munkireport/security/blob/b2683d8429306a3163b6691915005ba4a797c08c/scripts/security.py#L266-L269

```Error: No such key: NestedGroups
There was an unknown error.
Traceback (most recent call last):
  File "/usr/local/munki/preflight.d/security.py", line 385, in <module>
    main()
  File "/usr/local/munki/preflight.d/security.py", line 375, in main
    result.update({'ard_groups': ard_group_access_check()})
  File "/usr/local/munki/preflight.d/security.py", line 268, in ard_group_access_check
    ard_group = grp.getgrgid(group_id_out.split()[1]).gr_name
IndexError: list index out of range
```

**Comment by rickheil** [comment]
@bochoven I think this one can be closed as it was resolved in #2 yes?

**Comment by tuxudo** [comment]
Fixed in recent PRs
